src/applications/ivc-champva/10-10D/pages/ApplicantRelOriginPage.jsx

```diff
@@ -0,0 +1,73 @@
+import ApplicantRelationshipPage, {
```

Comment:
Custom page added - basically just a wrapper around `ApplicantRelationshipPage` but with a custom `generateOptions` function to give us the necessary custom wording for this screen.